src/main/scala/com/amazon/deequ/analyzers/Histogram.scala

```diff
@@ -43,7 +44,9 @@ case class Histogram(
     binningUdf: Option[UserDefinedFunction] = None,
     maxDetailBins: Integer = Histogram.MaximumAllowedDetailBins,
     where: Option[String] = None,
-    computeFrequenciesAsRatio: Boolean = true)
+    computeFrequenciesAsRatio: Boolean = true,
+    aggregateFunction: String = Histogram.Count,
```

Comment:
Can we make this an enum? In Scala 2, we can use a `sealed trait` and `case object` to create enumerations.